src/content/getting-started/architecture/broker/_index.en.md

```diff
@@ -9,8 +9,8 @@ basically a set of Custom Resource Definitions (CRDs) backed by the Kubernetes d
 The Broker also defines a ServiceAccount and RBAC components to enable other Submariner
 components to securely access the Broker's API.
 
-While there no Services associated with the Broker, if using ```subctl``` to deploy the Broker, an
-operator Pod is also deployed that installs the CRDs and the Globalnet configuration.
+While there are no services associated with the Broker, using ```subctl``` to deploy the Broker
+also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

Comment:
```suggestion
While there are no Services associated with the Broker, using ```subctl``` to deploy the Broker
also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

See [the Submariner docs style guide](https://submariner.io/development/website/style-guide/) and [the section on API objects in the Kubernetes docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).